README.md

```diff
@@ -13,6 +13,7 @@
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
+  - [💁 Video Presentation](#video-presentation)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Install](#install)

```

Comment:
- [OPTIONAL] Great work implementing the requirements for this project. However, you ought to have used _**GitFlow**_ for a solo capstone project and not the _**GitHub**_ flow. 

---

In _**Gitflow**_ for a solo capstone project, all implementation and commits ought to have been done in a feature branch intending to merge to the _**dev**_ branch as the base branch. The GitHub flow is for a group capstone project. 👍 😎